features/convert_config.feature

```diff
@@ -400,6 +400,24 @@ Feature: Convert config
       """
     And the temp "formatters.yaml" file should have been removed
 
+  Scenario: path options
```

Comment:
We test that the yaml config is correctly converted to the PHP config for this new option